Makefile

```diff
@@ -707,12 +708,9 @@ bdist_mpkg: 3rd
 fetch-data:
 	echo "I: fetching data from datadb"
 	[ -e datadb ] || mkdir -p datadb
-	rsync $(RSYNC_OPTS) $(DATA_URI)/tutorial_data $(DATA_URI)/mnist \
-		$(DATA_URI)/face_inversion_demo \
-	      	$(DATA_URI)/hyperalignment_tutorial_data \
-                $(DATA_URI)/haxby2001 \
-		datadb/ 
-	@for ds in datadb/*; do \
+	rsync $(RSYNC_OPTS) $(DATA_HOST)::'$(DATA_DIR)/tutorial_data $(DATA_DIR)/mnist $(DATA_DIR)/face_inversion_demo $(DATA_DIR)/hyperalignment_tutorial_data $(DATA_DIR)/haxby2001' \
```

Comment:
I tried (but failed) to split the part between single quotes at line 711 into multiple lines:: 

    '$(DATA_DIR)/tutorial_data $(DATA_DIR)/mnist $(DATA_DIR)/face_inversion_demo $(DATA_DIR)/hyperalignment_tutorial_data $(DATA_DIR)/haxby2001'

Any ideas how to do that, or is it just not possible?